recipes/genometools-genometools/build.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
-make
+
+
+make cairo=no
```

Comment:
When including cairo, the mulled test fails with 
gt: error while loading shared libraries: libXrender.so.1: cannot open shared object file: No such file or directory
(See for instance : https://travis-ci.org/Cashalow/bioconda-recipes/jobs/260260440)

So after adding xorg-libxrender, xorg-libxext and xorg-libsm (that are also needed, see https://travis-ci.org/Cashalow/bioconda-recipes/jobs/260260440 and https://travis-ci.org/Cashalow/bioconda-recipes/builds/260223795) the mulled-test still fails with 

gt: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory ( https://travis-ci.org/Cashalow/bioconda-recipes/jobs/260223797) 

which does not make a lot of sense to me. 

I tried also building genometools without using all the dependencies already bundled, which is the standard behavior, but I think I am way out of my league here. I think the problem is related to this discussion https://github.com/genometools/genometools/issues/742

Cairo is the only not dependency not bundled with genometools, I think that the problem comes from this but I'm not sure how to solve it. I think the previous build passed because at the time the mulled-test was not used ?